src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -102,6 +101,12 @@
 
     @Option(doc="Google Genomics API client_secrets.json file path.", common = true)
     public String GA4GH_CLIENT_SECRETS="client_secrets.json";
+
+    @Option(shortName = "jdk_deflater", doc = "Use the JDK Deflater instead of the IntelDeflater for writing BAMs", common = true)
```

Comment:
I think it would be better to use the same option name as GATK here, e.g. `use_jdk_deflater`, `use_jdk_inflater`.